### ISSUE TITLE
TypeError for routes returning arrays

### ISSUE THREAD

**Issue by thathoff** [distantnative/retour-for-kirby#275]
## Describe the bug

The `route:after` closure expects the 4th argument (`$result`) to be a `String`.  But in some cases Kirby does not insert a string here. This happens for example if you have a custom route returning an array.

## Steps to reproduce

1. Install Retour 4.0.0.beta-1
2. Setup a custom route returning an array like this
```php
[
    'pattern' => 'retourtest',
    'action' => function () {
        return ['foo' => 'bar'];
     }
],
```
3. Visit that route
4. TypeError `Argument 4 passed to Kirby\Cms\App::distantnative\Retour\{closure}() must be of the type string or null, array given` is thrown.

## Expected behavior

No TypeError thrown.

  - Retour version: 4.0.0-beta.1
  - Kirby version: 3.6.1

**Comment by iskrisis** [comment]
Came to report this - fix worked for me.

**Comment by distantnative** [comment]
Great, will try to release a new beta this weekend

**Comment by distantnative** [comment]
✅ Will be fixed in 4.0 release
